Cargo.toml.in

```diff
@@ -49,10 +49,10 @@ log = "0.4.17"
 rand = "0.8.5"
 spin = "0.9.5"
 # shared-memory enabled for zenoh even if zenoh-c "shared-memory" feature is disabled. This is to make "std::mem::transmute" work for `ZSLice`
-zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main", features = [ "shared-memory", "unstable" ], default-features = false }
```

Comment:
Should we keep the `version` in the `Cargo.toml.in`?

Reply:
I suppose that in the `main` branch the version should correspond to current state of main branch of other projects, The version field makes sense to set for tagged branches, to link them to correspponding dependencies.

Reply:
Correct, but cargo will fallback to `git` in case the `version` is not found. 
By doing so, it's easier for the release scripts to bump to the right version once available on `crates.io`. 
Better keep it then.

Reply:
To which version should we point it then? I've set "1.0.0-dev", correct?